packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -5,10 +5,10 @@
 /* eslint-disable max-lines */
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 /* eslint-disable @typescript-eslint/consistent-type-assertions */
-/* eslint-disable @typescript-eslint/no-shadow */
+
```

Comment:
Why the empty lines in this file?

Reply:
because i ran auto-fix which remove un-needed lint disables, but didn't remove the lines